pkg/ui/react-app/src/pages/graph/ExpressionInput.test.tsx

```diff
@@ -1,28 +1,15 @@
 import * as React from 'react';
 import { mount, ReactWrapper } from 'enzyme';
 import ExpressionInput from './ExpressionInput';
-import Downshift from 'downshift';
-import { Button, InputGroup, InputGroupAddon, Input } from 'reactstrap';
+import { Button, InputGroup, InputGroupAddon } from 'reactstrap';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faSearch, faSpinner } from '@fortawesome/free-solid-svg-icons';
 
-const getKeyEvent = (key: string): React.KeyboardEvent<HTMLInputElement> =>
-  ({
-    key,
-    nativeEvent: {},
-    preventDefault: () => {
-      // Do nothing.
-    },
-  } as React.KeyboardEvent<HTMLInputElement>);
-
-describe('ExpressionInput', () => {
-  const metricNames = ['instance:node_cpu_utilisation:rate1m', 'node_cpu_guest_seconds_total', 'node_cpu_seconds_total'];
+describe('CMExpressionInput', () => {
```

Comment:
Just a nit, for staying consistent with component name.
```suggestion
describe('ExpressionInput', () => {
```